### PR TITLE
Upstream animation-effect-timing-defaults.html from Blink

### PR DESCRIPTION
**Pull request by alancutter**
Import https://chromium.googlesource.com/chromium/src/+/ad55ab057af4360e0af892a0a1d3364925d8f970/third_party/WebKit/LayoutTests/web-animations-api/animation-effect-timing-defaults.html

### CODE REVIEW

File: web-animations/interfaces/AnimationEffectTiming/delay.html [hunk 1]
```diff
@@ -14,6 +14,12 @@
 
 test(function(t) {
   var div = createDiv(t);
+  var anim = div.animate({});
```
Comment (suzyh) [inline web-animations/interfaces/AnimationEffectTiming/delay.html]: animate(null)?

**Comment by suzyh** [comment]
lgtm

File: web-animations/interfaces/AnimationEffectTiming/delay.html (hunk 1 above)
Comment (alancutter) [inline web-animations/interfaces/AnimationEffectTiming/delay.html]: null would mean anim.effect is null.

Comment (alancutter) [inline web-animations/interfaces/AnimationEffectTiming/delay.html]: Changed to use null.

Comment (suzyh) [inline web-animations/interfaces/AnimationEffectTiming/delay.html]: Looks like delay and easing got missed?

Comment (alancutter) [inline web-animations/interfaces/AnimationEffectTiming/delay.html]: Fixed.

**Pull request merged**